alerts/src/androidLibMain/kotlin/AlertPresenter.kt

```diff
@@ -40,19 +44,25 @@ actual class AlertPresenter(
     actual class Builder(
         private val lifecycleManagerObserver: LifecycleManagerObserver = LifecycleManagerObserver()
     ) : BaseAlertPresenter.Builder(), LifecycleSubscribable by lifecycleManagerObserver {
-        actual override fun create(coroutineScope: CoroutineScope) = AlertPresenter(createAlert(), lifecycleManagerObserver, coroutineScope)
+        actual override fun create(coroutineScope: CoroutineScope) =
+            AlertPresenter(createAlert(), lifecycleManagerObserver, coroutineScope)
     }
 
     private companion object {
-        fun transform(style: Alert.Action.Style): Int = when (style) {
+        fun transform(alertStyle: Alert.Action.Style): Int = when (alertStyle) {
```

Comment:
new parameter name is misleading. this is actually a **button/action** style

Reply:
Did not notice that change, probably a rename picked it. Thanks.